### ISSUE TITLE
Clarify and document reviewing process for user contributed scripts and pipelines

### ISSUE THREAD

**Issue by jmlord** [GEO-BON/bon-in-a-box-pipelines#135]

**Comment by jmlord** [comment]
Jory and I will come to this eventually, but if someone knowledgeable of the project wants to make a first draft, it's most welcome!

**Comment by tpoisot** [comment]
I think something like [Freckleton (2018)](https://github.com/user-attachments/files/17333788/Freckleton.2018.-.Accessibility.reusability.reliability.-.Improving.the.standards.for.publishing.code.in.Methods.in.Ecology.and.Evolution.pdf) + the checklist that the editorial board came up with would be good.

If I may make a suggestion: PR template in addition to / instead of a qmd document?

(this also counts as me volunteering to assist with this)

**Comment by jmlord** [comment]
> If I may make a suggestion: PR template in addition to / instead of a qmd document?

Yes! I would see a PR template with a checklist and referring to that more complete documentation.

> (this also counts as me volunteering to assist with this)

That is most welcome :)

**Comment by jmlord** [comment]
@JoryGriffith has already drafted a standards document that will partly address this issue. It should be circulated soon among closer collaborators like you Tim and then go into a PR.
